lab/lablib/SoundManager.js

```diff
@@ -90,14 +93,15 @@ export class SoundManager {
     this.synths.poly = poly;
   }
 
-  init_patterns() {
+  init_scores() {
```

Comment:
The code to create these patterns should go in the sketch, `SoundManager` should just have a "register_score" method to compile and save the score.